.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.py]
+indent_size = 4
+
+[*.md]
+indent_size = 4
+trim_trailing_whitespace = false
```

Comment:
Curious why trailing whitespace would be something you'd want in .md files.  No opinion about yes or no, but just wondering the use case.

Reply:
I don't remember why this wan in there but glad to take it out (can always be added back if there's a need)